### DISCUSSION TITLE
How to rotate a line?

### DISCUSSION THREAD

**Discussion by madjxatw** [jgraph/drawio#4329] [Q&A]
I need to draw two orthogonal straight lines, but could not figure out how to rotate a line with an expected angle. Please advice, thx!

**Comment by psyrus** [comment]
It's a little hacky but you can create a rectangle and set the line size to 0.5 pt, collapse it down so that it appears as a line, and then you can rotate it as you would like to. You can see the little rotate icon at the bottom right here:

<img width="693" alt="image" src="https://github.com/jgraph/drawio/assets/8152287/743ffb84-b56b-4248-a5c7-591b75b63194">

**Comment by madjxatw** [comment]
I've also used the similar trick, but appreciate your advice. There seems to be no simple way yet in draw.io to do such.
